keyboards/j80/info.json

```
@@ -0,0 +1,202 @@
+{
+  "keyboard_name": "J80",
+  "url": "",
+  "maintainer": "JER",
```

This should actually be your GitHub username, if you're listing yourself as the keyboard maintainer in the readme file.

```suggestion
  "maintainer": "oeywil",
```